README.md

```diff
@@ -4,16 +4,38 @@ Node.js NAN C++ NuoDB Driver
 
 The node-nuodb add-on for Node.js powers high performance NuoDB Database applications.
 
-Use node-nuodb to connect Node.js 8 and 10 to NuoDB Database.
+Use node-nuodb to connect Node.js to a NuoDB Database.
```

Comment:
Did you remove the version information and not replace it with a specific known version that works?  Or was the point to list the dependency after...

Reply:
The point was to just have it in the dependency list.